ts/components/wallet/TransactionsList.tsx

```diff
@@ -106,6 +106,22 @@ export default class TransactionsList extends React.Component<Props, State> {
         text2={datetime}
         text3={paymentReason}
         onPressItem={() => this.props.navigateToTransactionDetails(item)}
+        accessible={true}
+        accessibilityRole={"button"}
+        accessibilityLabel={I18n.t(
+          "wallet.accessibility.transactionListItem.label",
+          {
+            payment: isNew
+              ? I18n.t(
+                  "wallet.accessibility.transactionListItem.payment.unread"
+                )
+              : I18n.t("wallet.accessibility.transactionListItem.payment.read"),
+            merchant: recipient,
+            amount,
+            datetime,
```

Comment:
```suggestion
            dateToAccessibilityReadbleFormat(datetime),
```
Could [it be useful](https://github.com/pagopa/io-app/blob/03420b24812fb4c826ccb6663b33dcdcdbf9be94/ts/utils/accessibility.ts#L65)?

Reply:
Since this would not read the time of the transaction i just made an improvement on `dateToAccessibilityReadableFormat`, you can see it in 8b1af1f